config/log4j2.properties

```
@@ -134,6 +134,10 @@ logger.slowlog.additivity = false
 logger.licensereader.name = logstash.licensechecker.licensereader
 logger.licensereader.level = error
 
+# Silence http-client by default
+logger.http_client.name = org.apache.http
+logger.http_client.level = fatal
```

It wouldn't hurt to be more explicit.

```suggestion
logger.apache_http_client.name = org.apache.http
logger.apache_http_client.level = fatal
```